src/main/java/com/slcube/shop/common/config/jpa/LoginUserAuditorAware.java

```diff
@@ -19,6 +21,11 @@ public Optional<String> getCurrentAuditor() {
             return null;
         }
 
+
+        if (authentication.getPrincipal().equals(ANONYMOUS_USER.getMemberStatus())) {
+            return Optional.ofNullable(ANONYMOUS_USER.getMemberStatus());
+        }
```

Comment:
요건 무조건 not null인걸로 보여지는데 그냥 Optional없이 쓰는게 어떨까요?? 새로운 개발자가 보면 null 가능성 때문에 null check도 필요해 보여서 코드 가독성에 해칠거같아서요
```suggestion
        if (authentication.getPrincipal().equals(ANONYMOUS_USER.getMemberStatus())) {
            return ANONYMOUS_USER.getMemberStatus();
        }
```

만약 다른 곳에서 Opional을 반환 해야하면 그때는 `Optional.of()` method를 추천 드립니다